semgrep_output_v1.atd

```diff
@@ -909,8 +909,8 @@ type tag = string
 
 (* Response by the backend to the CLI to the POST /scans/<scan_id>/config *)
 type scan_config = {
+    (* TODO: remove deployment_id + deployment_name from here, instead this will be in scan_response *)
```

Comment:
you mean scan_info?

Reply:
Yeah bit of both - in `scan_response.scan_info`